src/main/java/core/basesyntax/strategy/handlers/BalanceHandler.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.strategy.handlers;
+
+import core.basesyntax.db.Storage;
+
+public class BalanceHandler implements OperationHandler {
```

Comment:
fix name for all hendlers 
```suggestion
public class BalanceOperationHandler implements OperationHandler {
```